actors/src/main/kotlin/ktx/actors/events.kt

```diff
@@ -80,6 +80,89 @@ inline fun <Widget : Actor> Widget.onClickEvent(
   return clickListener
 }
 
+/**
+ * Attaches a [ClickListener] to this actor.
+ * @param listener invoked each time this actor is touched.
+ * @return [ClickListener] instance.
+ */
+inline fun Actor.onTouchDown(crossinline listener: () -> Boolean): ClickListener {
```

Comment:
Could we overload this method to consume a lambda with `Unit` return type? This would simplify the API for the users.

Reply:
it would be possible yes and I would suggest to return true, otherwise touchup will never be called. 
I don't know if there is anyone out there who would return false for that method. TBH I don't really understand it. In my opinion in this context you always want to return true no matter what.

Shall I just do that to make it simpler?

Reply:
> I don't know if there is anyone out there who would return false for that method.

I don't fully remember, but AFAIK it seems like this is either to stop other listeners from handling the event or to stop `touchUp` from firing. A niche use either way. If it's not trivial to add an overload, I think we should just go with the `Unit` lambda and let the users handle niche cases with custom listeners.

> With that we could also change `onTouchEvent` to only take one lambda and the user could the just check for `event.type` if it is touchDown or touchUp - what do you think?

Agreed, a simplified, single-lambda `onTouchEvent` could be useful.

Reply:
> I don't know if there is anyone out there who would return false for that method.

I don't fully remember, but AFAIK it seems like this is either to stop other listeners from handling the event or to stop `touchUp` from firing. A niche use either way. If it's not trivial to add an overload, I think we should just go with the `Unit` lambda and let the users handle niche cases with custom listeners.

> With that we could also change `onTouchEvent` to only take one lambda and the user could the just check for `event.type` if it is touchDown or touchUp - what do you think?

Agreed, a simplified, single-lambda `onTouchEvent` could be useful.

Reply:
Added now also single lambda extensions for all three cases